poker-hand-evaluator/src/evaluator/DeckOfCards.java

```diff
@@ -69,7 +52,7 @@ private void deleteFirstCardOfDeck() {
 
     @Override
     public String toString() {
-        return "playCards{" +
+        return "\nplayCards{" +
```

Comment:
i wouldn't put these \n-s in the beginning of your `toStrings` or `String.formats`